wine-tkg-git/PKGBUILD

```diff
@@ -1165,6 +1179,16 @@ package() {
 	  msg2 '##########################################################################################################################'
 	fi
 
+	if [ "$_dxvk_async" == "true" ]; then
```

Comment:
You'd want to also check that `$_use_dxvk` value is `true`

Reply:
Done.